core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/cluster/ClusterToClusterV4ResponseConverter.java

```diff
@@ -103,6 +103,11 @@ public ClusterV4Response convert(Cluster source) {
         return clusterResponse;
     }
 
+    // used for test
+    public void setCloudStorageConverter(CloudStorageConverter cloudStorageConverter) {
```

Comment:
in this case, please use with package protected

Reply:
done